aptos-move/framework/aptos-framework/sources/Coin.move

```diff
@@ -608,6 +608,11 @@ module AptosFramework::Coin {
         destroy_zero(zero);
     }
 
+    #[test_only]
+    public fun destroy_mint_cap<CoinType>(mint_cap: MintCapability<CoinType>) {
+        let MintCapability<CoinType> { } = mint_cap;
+    }
+
```

Comment:
why here? why not another pr or commit

Reply:
Because a test in AptosGovernance needs to destroy mint capability (cannot be dropped)